docs/v3.x/getting-started/installation.md

```diff
@@ -1,6 +1,12 @@
-# ⚙️ Installation
+# Installation
 
-### Installation guides
+Strapi gives you many possible installation options for your project or application. Strapi can be installed on your computer or services such as Digital Ocean or Platform.sh. The following documentation covers differents options to install strapi and getting sttarted.
+
+::: tip
+For a more detailed overview of deployment please see the [related documentation](./deployment).
```

Comment:
Not sure if this needs a .html or not, since we have so many issues with other links.

Reply:
Need to add `.md` you are right